angular.json

```diff
@@ -61,7 +61,7 @@
                 {
                   "type": "initial",
                   "maximumWarning": "6mb",
-                  "maximumError": "8mb"
+                  "maximumError": "9mb"
```

Comment:
This is somewhat concerning. Is the bundle size really above 8MB now? 😱

I think this might need looking at during PI week, @Mo-Lala84.

Reply:
Yes, this considered as tech debt given limited time for todays release.